images/openwisp_dashboard/load_init_data.py

```diff
@@ -254,6 +254,6 @@ def create_default_topology(vpn):
     create_default_credentials()
     create_ssh_key_template()
 
-    if os.environ.get('USE_OPENWISP_TOPOLOGY', False):
+    if os.environ.get('USE_OPENWISP_TOPOLOGY', False) == "True":
```

Comment:
Can we use `env_bool` here instead? So, it will be consistent with the rest of the code.

```suggestion
    if env_bool(os.environ.get('USE_OPENWISP_TOPOLOGY')):
```

Here's an example:

https://github.com/openwisp/docker-openwisp/blob/f8156d6371798ec6c439fc5ff83a05832d5c8074/images/common/openwisp/asgi.py#L24-L27

Reply:
Yes of course ! I'll edit the PR tomorrow with this change.